plugins/flytekit-whylogs/flytekitplugins/whylogs/transformer.py

```diff
@@ -0,0 +1,49 @@
+from typing import Type
+
+from whylogs.core import DatasetProfileView
+
+from flytekit import FlyteContext, BlobType
+from flytekit.extend import T, TypeTransformer, TypeEngine
+from flytekit.models.literals import Literal, Scalar, Blob, BlobMetadata
+from flytekit.models.types import LiteralType
+
+
+class WhylogsDatasetProfileTransformer(TypeTransformer[DatasetProfileView]):
+    """
+    Transforms whylogs Dataset Profiles to and from a Schema (typed/untyped)
+    """
+
+    # TODO Spark plugin is a SchemaType. How do I pick? Is blob right?
```

Comment:
blob is right if you just want a file yeah.  The SchemaType is kinda deprecated.  It's still supported, but for columnar data we wrote the StructuredDataset type to handle dataframes.  but this is a protobuf message we're talking about right?  not a dataframe.